src/redux/greeting/greeting.js

```diff
@@ -19,7 +19,7 @@ export const getMessage = (payload) => ({
 });
 
 export const getGreeting = () => async (dispatch) => {
-  const response = await fetch('http://127.0.0.1:3000/v1/greeting.json');
+  const response = await fetch('http://127.0.0.1:8000/v1/greeting');
```

Comment:
- [ ] I cant see anything on the page. It gives me an error. Kİndly fix it. Also please don't merge the branch before review right now ı can see only a few files. Thats why ı cant give correct inline comments.

![image](https://user-images.githubusercontent.com/38635158/180606879-4439241c-3337-47be-a75a-6b25a6bdc6d3.png)